docs/CHANGELOG.rst

```diff
@@ -29,6 +29,7 @@ Added
 - File upload processes
 - Resolwe process tests
 - Add SET_ENV setting to set environment variables in executor
+- Support file lists in archiver
```

Comment:
I think this is unnecessary since the change log for this release already has `Archive file process` under _Added_